seacrowd/sea_datasets/unisent/unisent.py

```diff
@@ -0,0 +1,286 @@
+    "aoz",
+    "atb",
+    "atd",
+    "att",
+    "ban",
+    "bbc",
+    "bcl",
+    "bgr",
+    "bgs",
+    "bgz",
+    "bhp",
+    "bkd",
+    "bku",
+    "blw",
+    "blz",
+    "bpr",
+    "bps",
+    "bru",
+    "btd",
+    "bth",
+    "bto",
+    "bts",
+    "btx",
+    "bug",
+    "bvz",
+    "bzi",
+    "cbk",
+    "ceb",
+    "cfm",
+    "cgc",
+    "clu",
+    "cmo",
+    "cnh",
+    "cnw",
+    "csy",
+    "ctd",
+    "czt",
+    "dgc",
+    "dtp",
+    "due",
+    "duo",
+    "ebk",
+    "fil",
+    "gbi",
+    "gor",
+    "heg",
+    "hil",
+    "hnj",
+    "hnn",
+    "hvn",
+    "iba",
+    "ifa",
+    "ifb",
+    "ifk",
+    "ifu",
+    "ify",
+    "ilo",
+    "ind",
+    "iry",
+    "isd",
+    "itv",
+    "ium",
+    "ivb",
+    "ivv",
+    "jav",
+    "jra",
+    "kac",
+    "khm",
+    "kix",
+    "kje",
+    "kmk",
+    "kne",
+    "kqe",
+    "krj",
+    "ksc",
+    "ksw",
+    "kxm",
+    "lao",
+    "lbk",
+    "lew",
+    "lex",
+    "lhi",
+    "lhu",
+    "ljp",
+    "lus",
+    "mad",
+    "mak",
+    "mbb",
+    "mbd",
+    "mbf",
+    "mbi",
+    "mbs",
+    "mbt",
+    "mej",
+    "mkn",
+    "mnb",
+    "mog",
+    "mqj",
+    "mqy",
+    "mrw",
+    "msb",
+    "msk",
+    "msm",
+    "mta",
+    "mtg",
+    "mtj",
+    "mvp",
+    "mwq",
+    "mwv",
+    "mya",
+    "nbe",
+    "nfa",
+    "nia",
+    "nij",
+    "nlc",
+    "npy",
+    "obo",
+    "pag",
+    "pam",
+    "plw",
+    "pmf",
+    "pne",
+    "ppk",
+    "prf",
+    "prk",
+    "ptu",
+    "pww",
+    "sas",
+    "sbl",
+    "sda",
+    "sgb",
+    "smk",
+    "sml",
+    "sun",
+    "sxn",
+    "szb",
+    "tbl",
+    "tby",
+    "tcz",
+    "tdt",
+    "tgl",
+    "tha",
+    "tih",
+    "tlb",
+    "twu",
+    "urk",
+    "vie",
+    "war",
+    "whk",
+    "wrs",
+    "xbr",
+    "yli",
+    "yva",
+    "zom",
+    "zyp",
+    "pse",
+    "mnx",
+    "mmn",
+    "lsi",
+    "hlt",
+    "gdg",
+    "bnj",
+    "acn",
+]
+
+_LICENSE = Licenses.CC_BY_NC_ND_4_0.value  # cc-by-nc-nd-4.0
+_LOCAL = False
+_URLS = {
+    _DATASETNAME: {lang: URL.format(lang) for lang in _LANGUAGES},
+}
+
+_SUPPORTED_TASKS = [Tasks.SENTIMENT_ANALYSIS]
+
+_SOURCE_VERSION = "1.0.0"
+_SEACROWD_VERSION = "1.0.0"
+
+
+class UniSentDataset(datasets.GeneratorBasedBuilder):
+    LABELS = ["NEGATIVE", "POSITIVE"]
+
+    BUILDER_CONFIGS = [SEACrowdConfig(name=f"{_DATASETNAME}_{lang}_source", version=datasets.Version(_SOURCE_VERSION), description=_DESCRIPTION, schema="source", subset_id=f"{_DATASETNAME}_{lang}",) for lang in _LANGUAGES] + [
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_{lang}_seacrowd_text",
+            version=datasets.Version(_SEACROWD_VERSION),
+            description=_DESCRIPTION,
+            schema="seacrowd_text",
+            subset_id=f"{_DATASETNAME}_{lang}",
+        )
+        for lang in _LANGUAGES
+    ]
+
+    DEFAULT_CONFIG_NAME = f"{_DATASETNAME}_source"
+
+    def _info(self) -> datasets.DatasetInfo:
+        features = None
```

Comment:
any justifications to pre-add it as none? why don't just throw an exception on unexpected `self.config.schema`?